### ISSUE TITLE
As the number of iterations increases, the image quality decreases

### ISSUE THREAD

**Issue by Zhou-Yijie** [luosiallen/latent-consistency-model#96]
I trained the Dreamshaper_v7 model following the settings of train_lcm_distill_sd_wds.py. In the early stages of training (such as 100 iterations), the output image looks like this:
![image](https://github.com/luosiallen/latent-consistency-model/assets/118658953/7ef4512e-63e7-437d-96f7-b0d025736ab1)
But as the number of iterations increases (such as 1000iteration), the output image becomes low quality and rough, like this:
![image](https://github.com/luosiallen/latent-consistency-model/assets/118658953/fad72c17-df76-4951-987b-edde01775659)
More iterations did not fix it.
Has anyone encountered such a problem?

**Comment by yuyongcan** [comment]
Have you solved this?
When I tried to train the SD2.1 model on my own datasets, I found that the loss didn't converge and is higher than the initial iteration.

**Comment by Tramac** [comment]
+1

**Comment by mk322** [comment]
+1 I faced the same issue on video generation.

**Comment by dragon-cao** [comment]
+1 I faced the same issue.Have you solved this?

**Comment by yangzhenyu6** [comment]
Hello, I also trained the Dreamshaper-v7 model according to the settings of train_1cm_istill_std_wds.comy, but in the end, I got pytorch-lora_ weights. safetensors. How can I call them to see the effect? I tried various methods but couldn't call them. How did you see the effect in the picture?
